app/assets/stylesheets/application.scss

```diff
@@ -33,9 +33,9 @@
 }
 
 .dashed {
-  border: 2px dashed #808080;
-  height: 93%;
-  width: 93%;
+   border: 2px dashed gray;
```

Comment:
Color `gray` should be written in hexadecimal form as `#808080`<br>Color literals like `gray` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 3 spaces